x/authenticator/integration_test.go

```diff
@@ -315,26 +316,25 @@ func (s *AuthenticatorSuite) TestAuthenticatorStateExperiment() {
 		Amount:      sdk.NewCoins(sdk.NewInt64Coin(sdk.DefaultBondDenom, 1_000_000_000_000)),
 	}
 
-	stateful := testutils.StatefulAuthenticator{KvStoreKey: s.app.GetKVStoreKey()[authenticatortypes.StoreKey]}
+	stateful := testutils.StatefulAuthenticator{KvStoreKey: s.app.GetKVStoreKey()[authenticatortypes.ManagerStoreKey]}
```

Comment:
```suggestion
	stateful := testutils.StatefulAuthenticator{KvStoreKey: s.app.GetKVStoreKey()[authenticatortypes.AuthenticatorStoreKey]}
```

This is the reason the test is failing

Reply:
fixed :bow: